src/odemis/driver/orsay.py

```diff
@@ -584,7 +635,7 @@ def _changeVacuum(self, goal):
         logging.debug("Setting vacuum status to %s." % self.axes["vacuum"].choices[goal])
         self._vacuumStatusReached.clear()  # to make sure it will wait
         self._chamber.VacuumStatus.Target = goal
-        if not self._vacuumStatusReached.wait(18000):  # wait maximally 5 hours
+        if not self._vacuumStatusReached.wait(18000):  # wait maximally 5 hours  TODO: TUNE THIS?
```

Comment:
As a user if I need to wait 5 hours on something I would like to have a least a regular feedback in the log that the whole system isn't freezed but actively pumping or something.
Is such a message/pop-up or something provided if pumping takes more than 10 minutes?

Reply:
This issue is solved in another pending pull request, with all results from the hardware tests done. Basically, it generally only takes 10 minutes, so it waits maximally 30 minutes. In the mean time values like chamber pressure, pumping speed etc. are kept up to date, so the user should be able to see those values change and so know the program did not freeze.